enterprise/server/cmd/ci_runner/main.go

```diff
@@ -832,6 +833,10 @@ func run() error {
 // to be called after the current invocation has completed, to avoid blocking
 // the invocation status from being reported.
 func (ws *workspace) prepareRunnerForNextInvocation(ctx context.Context, taskWorkspaceDir string) {
+	if *skipAutomaticConfiguration {
+		return
+	}
```

Comment:
this doesn't feel the same as automatic configuration, since it's cleanup that maybe voids the runner or cleans it up. Do we really need to skip here in this case?

Reply:
If we don't run our automatic setup, we can't guarantee the user cloned a git repo or even if they do, if it's at the path we expect. So the git cleanup and bazel checks we run may / likely will fail